platform/windows/display_server_windows.cpp

```diff
@@ -5787,6 +5787,8 @@ void DisplayServerWindows::_process_key_events() {
 					if (!(ke.lParam & (1 << 24)) && ToUnicodeEx(extended_code, (ke.lParam >> 16) & 0xFF, keyboard_state, chars, 255, 4, GetKeyboardLayout(0)) > 0) {
 						String keysym = String::utf16((char16_t *)chars, 255);
 						if (!keysym.is_empty()) {
+							Key unicode_keysym = KeyMappingWindows::get_unicode_keysym(keysym[0]);
```

Comment:
ASCII values are mapped to Godot key constants exactly, there's no need for the mapping:

```suggestion
							Key unicode_keysym = (Key)keysym[0];
```

Reply:
Thank you.
Indeed, when Godot's key codes match ASCII key codes, no lookup table is needed.